src/framework/preferences/css/stylus/Enactors.styl

```diff
@@ -25,9 +25,11 @@
     font-family: "Comic Sans MS", sans-serif !important;
 }
 
-// UI Enhancer "Enlarge text" link options
+// UI Enhancer "Enlarge text" input/button options
 .fl-text-larger input,
-.fl-text-larger button {
+.fl-text-larger button,
+.fl-text-larger .fl-input-enhance,
+.fl-text-larger .fl-button-enhance, {
```

Comment:
Umm... I wonder why integrator's custom made buttons will use a css name of `fl-button-enhance`. Is this name suggested somewhere in our docs?